.env.example

```diff
@@ -3,14 +3,13 @@
 # Important variables
 BASE_URL=appserver.test
 
-#we wish :(
-SHORTENER_DOMAIN=aeg.ee
+SHORTENER_DOMAIN=myaeg.ee
 
 EMAIL_DOMAIN=
 
 # To enable services, add them (i.e. the name of enclosing folder) to the array, like
 #ENABLED_SERVICES=gateways:frontend:core:statutory:events:network:summeruniversity:discounts:knowledge:mailer:dev-tools
-ENABLED_SERVICES=gateways:frontend:core:statutory:events:summeruniversity:discounts:mailer
+ENABLED_SERVICES=gateways:frontend:core:statutory:events:summeruniversity:discounts
```

Comment:
Mailer should be enabled in some way otherwise all processes that try to send an email will fail

Reply:
I wanted to work on that on a future MR (i did notice that i couldn't even add to a group without it to fail..!)

Reply:
my priority is to make the setup smooth, i think it is better to have that although with an incomplete system

Reply:
I prefer to still have mailer in here so we don't forget to re-add it for new people. I'll take a look at the mailer error soon

Reply:
I don't think we will forget as we have plenty of capacity for mentoring 1-1 newcomers, they just don't ask questions and disappear after a while :/ but i added it again although i am very against having a broken installation

Reply:
I think I might have fixed it, did you see my mailer PR?

Reply:
i'll look at it now!

Reply:
it works, i will bump the version of the submodule once merged on your side

Reply:
PR has been merged and submodule has been bumped by dependabot